draft-ietf-quic-transport.md

```diff
@@ -788,6 +793,29 @@ Implementations MUST assume that an unsupported version uses an unknown packet
 format. All other fields MUST be ignored when processing a packet that contains
 an unsupported version.
 
+## The Latency Spin Bit {#spin-bit}
+
+The latency spin bit enables latency monitoring from observation points on the
+network path. Each endpoint, client and server, maintains a spin value, 0 or
+1, for each QUIC connection, and sets the spin bit on packets it sends for
+that connection to the appropriate value. It also maintains the highest packet
+number seen from its peer on the connection. The value is then determined at
+each endpoint as follows:
+
+* The server initializes its spin value to 0. When it receives a packet from
```

Comment:
I would probably start with the client, since it initiates the process.  Also, it seems it doesn't matter what the server initializes to, since it always echoes the received client's value, so I would omit this sentence.

Reply:
Will change the order. 

However, since spinning only happens on the short header, I think it needs to be specified like this to handle the case that the server starts sending short header packets before the client does (e.g., because it's done sending all the non-ACK frames it needs to in the 0RTT part of the connection)